pkg/mail/mail.go

```diff
@@ -20,6 +20,42 @@ func Sendmail(to string, emailSubject string, emailBody string) error {
 	m.SetHeader("Subject", emailSubject)
 	m.SetBody("text/plain", emailBody)
 
+	err := Sendmail(m)
+	if err != nil {
+		return err
+	}
+
+	log.Debugf("Email to %s was sent", to)
+
+	return nil
+}
+
+func SendMailWithFrom(to string, emailSubject string, emailBody string, from string) error {
+
+	m := gomail.NewMessage()
+
+	mailConfig := config.GetMailConfig()
+
+	m.SetHeader("From", mailConfig.SMTPFrom)
+	m.SetHeader("Reply-To", from)
```

Comment:
Why do you call the option `from` when it is added to `Reply-to`? I would also unify both functions and check here if `from != ""`.

Reply:
In your own local mail config e.g. Google, it says, that the sender (`From`) cannot be verified. I think this happens, when the `From` differs from the actual SMTP login/server. To prevent mails from landing in the Junk Folder I used the `Reply-To` instead.

Reply:
Ah, ok great :D